widgets/src/main/java/org/pentaho/mantle/client/dialogs/folderchooser/SelectFolderDialog.java

```diff
@@ -63,18 +63,21 @@ public void onBrowserEvent( Event event ) {
     }
   }
 
-  private static MySolutionTree tree = new MySolutionTree( false );
+  private static final MySolutionTree tree = new MySolutionTree( false );
 
-  public SelectFolderDialog() {
+  private String defaultSelectedPath = FolderTree.getHomeFolder();
+
+  public SelectFolderDialog( String selectedPath ) {
```

Comment:
Ideally, we'd add a new constructor and would thus not break compatibility. While you've updated all known Pentaho maintained references, it is never very clear what's public or not, and can be being used by third-parties or not.